client/src/App.js

```diff
@@ -51,6 +52,7 @@ function App() {
           <Route path="/signup" component={SignUp} />
           <Route path="/dealer" component={OrderProductsDealer} />
           <Route path="/manufacturer" component={OrderProductsManufacturer} />
+          <PrivateRoute path="/change/password" component={ChangePassword} />
```

Comment:
@ranjit977 any reason why it's private, not public?

Reply:
only signed in user can changed tha password. Meaning those who are logged in.

Reply:
@ranjit977 make it public since this endpoint must be available for everyone

Reply:
@sergust  there is no point in making it a public route. It is already available for everyone with the private route. Please mark it as resolved.